src/wrapper/data_chunk.mli

```diff
@@ -7,6 +7,13 @@ val get_exn : t -> 'a Type.Typed_non_null.t -> int -> 'a array
 val get_opt : t -> 'a Type.Typed.t -> int -> 'a option array
 val free : t -> here:Source_code_position.t -> unit
 
+(** Returns the number of columns in the data chunk. If count is provided, uses that value.
+    Otherwise, determines the count by checking vectors. *)
+val column_count : ?count:int -> t -> int
```

Comment:
The column count is just not a thing that data chunks have information on, so rather than trying to guess it, `to_string_hum` should always take it as an argument.